feature/staff/src/main/kotlin/io/github/droidkaigi/confsched2023/staff/section/StaffSheet.kt

```diff
@@ -0,0 +1,70 @@
+package io.github.droidkaigi.confsched2023.staff.section
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import io.github.droidkaigi.confsched2023.model.Staff
+import io.github.droidkaigi.confsched2023.staff.section.StaffSheetUiState.Loading
+import io.github.droidkaigi.confsched2023.staff.section.StaffSheetUiState.StaffList
+import kotlinx.collections.immutable.PersistentList
+
+internal sealed interface StaffSheetUiState {
+    data object Loading : StaffSheetUiState
+    data class StaffList(
+        val staffs: PersistentList<Staff>,
+    ) : StaffSheetUiState
+}
+
+@Composable
+internal fun StaffSheet(
+    uiState: StaffSheetUiState,
+    modifier: Modifier = Modifier,
+    onStaffClick: (url: String) -> Unit,
+) {
+    Box(modifier = modifier.fillMaxSize()) {
+        when (uiState) {
+            Loading -> LoadingContent(modifier = Modifier.fillMaxSize())
+            is StaffList -> StaffList(
+                staffs = uiState.staffs,
+                onStaffClick = onStaffClick,
+            )
+        }
+    }
+}
+
+// FIXME: It might be a good idea to move this Composable function to a common module somewhere. The same applies to TimetableLoadingContent as well.
+@Composable
+private fun LoadingContent(
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier,
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.spacedBy(
+            space = 16.dp,
+            alignment = Alignment.CenterVertically,
+        ),
+    ) {
+        CircularProgressIndicator()
+
+        Text(
+            modifier = Modifier.fillMaxWidth(),
+            text = "Loading...",
+            textAlign = TextAlign.Center,
+            style = MaterialTheme.typography.bodyMedium,
+            overflow = TextOverflow.Ellipsis,
+        )
+    }
+}
```

Comment:
(This comment is just a question for the reviewer 🙇‍♀️ ) Would it be possible in such case to move `TimetableLoadingContent` to `core/designsystem/component` and reuse it...? 🤔

Reply:
fixed [152a5c0](https://github.com/DroidKaigi/conference-app-2023/pull/730/commits/152a5c0afda7d55f12c13131b619df58ad62554a) [4f2aacc](https://github.com/DroidKaigi/conference-app-2023/pull/730/commits/4f2aaccd490c8155d1e7f6d451ed97c466024040)